Casks/zed-preview.rb

```diff
@@ -0,0 +1,30 @@
+cask "zed-preview" do
+  version "0.120.3"
+  sha256 "2c44a02b7667a324933e779488c3c70c949089741ee387b39cda463dd3f3558e"
+
+  url "https://zed.dev/api/releases/preview/#{version}/Zed.dmg"
+  name "Zed Preview"
+  desc "Multiplayer code editor"
+  homepage "https://zed.dev/"
+
+  livecheck do
+    url "https://zed.dev/releases/preview"
+    regex(%r{href=.*?/preview/(\d+(?:\.\d+)+)/Zed.dmg}i)
+  end
+
+  auto_updates true
+  conflicts_with cask: [
+    "zed",
+  ]
```

Comment:
```suggestion
  conflicts_with cask: "zed"
```

Just a small formatting nitpick here, that doesn't currently get picked up by